src/openforms/formio/components/custom.py

```diff
@@ -198,6 +198,16 @@ def rewrite_for_request(component, request: Request):
             component["initialCenter"]["lat"] = config.form_map_default_latitude
             component["initialCenter"]["lng"] = config.form_map_default_longitude
 
+        if component.get("backgroundIdentifier", False):
+            try:
+                leaflet_map_background = LeafletMapBackground.objects.get(
+                    identifier=component["backgroundIdentifier"]
+                )
+                # Write the background url information
+                component["url"] = leaflet_map_background.url
```

Comment:
Let's move this out from the try block and emphasize only on the line which will cause the exception.

Reply:
Yeah sounds good. Will do 👍